### ISSUE TITLE
Error message when running setup.py

### ISSUE THREAD

**Issue by tomcass240** [nmap/nmap#2413]
When trying to install zenmap I get the following error message:

Writing /usr/local/lib/python2.7/dist-packages/zenmap-7.92SVN.egg-info
Traceback (most recent call last):
  File "./setup.py", line 588, in <module>
    setup(**setup_args)
  File "/usr/lib/python2.7/distutils/core.py", line 151, in setup
    dist.run_commands()
  File "/usr/lib/python2.7/distutils/dist.py", line 953, in run_commands
    self.run_command(cmd)
  File "/usr/lib/python2.7/distutils/dist.py", line 972, in run_command
    cmd_obj.run()
  File "./setup.py", line 188, in run
    self.fix_paths()
  File "./setup.py", line 340, in fix_paths
    "CONFIG_DIR": os.path.join(self.saved_prefix, config_dir),
  File "/usr/lib/python2.7/posixpath.py", line 70, in join
    elif path == '' or path.endswith('/'):
AttributeError: 'NoneType' object has no attribute 'endswith'

I am running Debian bullseye.

**Comment by goarblt** [comment]
hello
https://www.evil.com/

**Comment by SkoZombie** [comment]
I have the same issue with Python 3.10.6 on Mint 21

**Comment by TheProdigyLeague** [comment]
In layman's terms: 
>>> from py
>>> import py_v-2.7 --mod
>>> pip zenmap-7.92 from SVN.egg-info
COMPILE ERROR:
`
parse/!\./usr/local/lib/python2.7/dist-packages/zenmap-7.92SVN.egg-info~#$ sudo apt install zenmap
>>> throw ERROR: Traceback (most recent call last):
"./setup.py"setup(**setup_args);
"/usr/lib/python2.7/distutils/core.py"
>>> dist.run_commands(nill)
"/usr/lib/python2.7/distutils/dist.py" 
>>> run_commands
>>> self.run_command(cmd)
"/usr/lib/python2.7/distutils/dist.py" 
>>> run_command
>>> cmd_obj.run(exe)
"./setup.py"
>>> self.fix_paths(NUL)
"./setup.py"
>>> in fix_paths
"CONFIG_DIR": os.path.join(self.saved_prefix, config_dir):
"/usr/lib/python2.7/posixpath.py"
>>> in join
    elif path == '' or path.endswith('/'):
        AttributeError: 'NoneType' object has no attribute 'endswith'
`
